docs/src/references/c8y-configuration-management.md

```diff
@@ -163,3 +275,115 @@ Note that:
 * Since the type of configuration file is used as an MQTT topic name, the characters `#` and `+` cannot be used in a type name.
   If such a character is used in a type name (or in the path of a configuration file without explicit type),
   then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
+
+## Configuration protocol between thin-edge and the child-devices
+
+The configuration plugin `c8y_configuration_plugin` can act as a proxy between the cloud and a child-device.
+However, for that to work, a client must be installed on the child device
+to perform the actual configuration updates pushed from the cloud.
```

Comment:
```suggestion
However, for that to work, a client must be installed on the child device
to perform the actual configuration updates pushed from the cloud via the thin-edge device.
```

Reply:
Changed to "pushed by thin-edge on behalf of the cloud".